dwave/cloud/client/base.py

```diff
@@ -439,6 +440,8 @@ def from_config(cls, config_file=None, profile=None, client=None, **kwargs):
     def __init__(self, **kwargs):
         logger.debug("Client init called with: %r", kwargs)
 
+        self._initialized = False
```

Comment:
This is so that if there is an error in the next block we're still left in a valid state? Is it possible to detect whether the client is currently initialized via other attribute(s) saved on the object?

Reply:
I wanted to make sure this flag is available in worker threads that are started before object finishes initializing. Ended up not needing the flag in workers, but still it's probably safer this way.